app/views/admin/users/index.html.erb

```diff
@@ -34,7 +34,7 @@
                 <%= user_image(user, variant: :large) %>
               </div>
               <div>
-                <div><%= user.soft_deleted? ? user.login : link_to(user.login, user_home_path(user.login)) %></div>
+                <div><%= user.soft_deleted? ? user.display_name : link_to(user.display_name, user_home_path(user.login)) %></div>
```

Comment:
Sorry, I should have been more clear here — the `link_to` still needs to use `login`, that's the unique key that we find by.

Reply:
Whups ignore that comment, I misread!